source/systems/MSU-HPC_user_guide.rst

```diff
@@ -1416,40 +1499,37 @@ find the email, search your emails with the following:
    required within 3 days. Security training must then be completed before HPC2
    resources can be accessed.
 
-   Visit https://taps.hpc.msstate.edu to complete these requirements.
+   Visit TAPS to complete these requirements.
 
 
-.. rubric:: Login to MSU's Training and Password System
+**Login to MSU's Training and Password System**
 
-- Within 3 days of receiving the email, navigate to
-  `<https://taps.hpc.msstate.edu>`__
+- Within 3 days of receiving the email, navigate to TAPS.
 - Authenticate using your username and your temporary password.
 
 .. note::
 
-   If your temporary 3-day password has expired, it will need to be reset. See:
-   `Logging In - Reset Password
-   <https://oriondocs.rdhpcs.noaa.gov/wiki/index.php/Logging_in#Password>`__
+   If your temporary 3-day password has expired, it will need to be reset. 
 
 -  Upon successful login, you will see the TAPS Home page.
 
-.. rubric:: Take MSU Security Training
+**Take MSU Security Training**
 
 -  Click on the IT Security *Start training* button.
 -  Upon successful completion of the training, you will get a confirmation.
 -  Go back to the TAPS Home page.
 
-.. rubric:: Take MSU Insider Threat Training
+**Take MSU Insider Threat Training**
 
 -  Click on the Insider Threat *Start training* button. Upon successful
    completion of the training, you will get a confirmation.
 -  Go back to the TAPS Home page.
 
-.. rubric:: Dual-factor authentication and Password Change (User)
+**Dual-factor authentication and Password Change (User)**
```

Comment:
Looking closer, this may be a step for the user to follow.  If so, then these shouldn't be "headings", but numbers in a list?  ATM I'm not sure, but they need to be something different than just bold text.